frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/storage-tab/vm-wizard-storage-modal-enhanced.tsx

```diff
@@ -80,6 +84,10 @@ const VMWizardStorageModal: React.FC<VMWizardStorageModalProps> = (props) => {
     },
   ];
 
+  const allowedBusses: Set<DiskBus> = templateValidations
```

Comment:
we can do the same trick here as well

`(templateValidations || new TemplateValidations()).getAllowedBusses()`